### ISSUE TITLE
Can't find BIOS and then have to reset the calculator

### ISSUE THREAD

**Issue by Graphics-Calculator-Gamer** [Vogtinator/gpsp#10]
I've got a problem with gpSP-Nspire where it works perfectly after resetting the calculator but then when I turn it off and then back on it tells me the BIOS is missing and I have to reset the to play TI-Nspire CX CAS again. This happens with all games.

**Comment by Vogtinator** [comment]
Where did you download gpsp from? Which calculator model and which OS version is it running on?

**Comment by Graphics-Calculator-Gamer** [comment]
I downloaded gpsp from TI-Planet. The calculator is a TI-Nspire CX CAS and is running OS 4.5.0.1180.

**Comment by Strata69** [comment]
Yes, I've also had this problem quite often (with both calc84maniac's gpSP v0.11 and Vogt's v0.91) and this is just speculation, but do you think it is in any way related to Ndless not running properly on my calc? for example, with the addition of error messages in Ndless 5.3 (good stuff!) I can see that the main reason my calculator used to crash (i opened the issue on the Ndless git regarding "Ndless 5.2.0 does not work well with N-0720AL models") is probs due to it being unable to find ndless_resources.tns (i can see this from the error messages). is it just a coincidence that everyone I know has either faced neither of these problems (and has an older calc revision) or faced both these issues (and has the same calc revision as me)? I just suspect that there is some sort of correlation as both these issues involve being unable to find a file that is obviously there and is also solvable by cutting and pasting the file back, but I have no concrete evidence of such. 

Sorry if this was a bit long-winded, I just thought of this and wanted to put it down in writing before I forgot anything :P Anyways, good luck with gpSP and Ndless Vogt, hope you continue making great programs for the Nspire!
